zowe_conformance/terms_and_conditions.md

```diff
@@ -67,16 +85,24 @@ If The Linux Foundation confirms to the Participant that the tests have been su
 
 The participant must continue to abide by the terms and conditions of the Conformance Program throughout the period of the Participant's use of the Program Marks. This includes all of the terms set forth in this agreement, including without limitation the following specific requirements:
 
-> *Conformance Test Versions*: Conformance for a Qualifying Offering is tied to a specific version of the corresponding conformance tests and evaluation criteria for one or more particular Components. The tests' and evaluation criteria version number and the corresponding Component must be specified within the Conformance Mark logos, as described in the Visual Branding Guide. When a new version of the Test Evaluation Guide is released, conformant applications and services must demonstrate that they pass the updated tests and meet the updated evaluation criteria within 60 days. Applications that do not do so within the allotted time must not be marked as conformant thereafter. A previously-Qualifying Offering can re-conform to the new Test Evaluation Guide to regain conformance at any time by following the steps described above.
+>
```

Comment:
@Joe-Winchester I'm trying to understand all the additions here. Can you elaborate on everything trying to be achieved here?

Reply:
@jmertic - in 2019 the conformance program had the ability to introduce a change and if previously conformant plugins didn't re-apply they lost their badge.  Feedback from extenders is that they didn't want to lose conformance.  In v1 conformance extenders get a badge.  If a program is then revised with new criteria that will be optional (so conformance is not lost) but there needed to be a way that for folks who get the updated criteria they could advertize this and the catalog of conformant plugins would likewise let an end user see which offerings had which additional minor revision of conformance.   For example,  in v1 conformance basic offerings can get the badge.  In June (for example) single-sign-on is added to the conformance criteria which is optional but has end user value.  If someone applies for this they can then they will get an updated badge with SSO sticker on it.  Offerings who don't apply won't lose v1 conformance.  MFA might be another sticker.  They are akin to boy scout merit badges for conformance revision, but they are not breaking and do not cross version boundaries.

Reply:
Gotcha - I think it would be helpful for me to be a part of the next Onboarding squad meeting to talk through this. I had thought there was a discussion to approach this a different way.

Reply:
@Joe-Winchester Friendly reminder to revert these changes when you have a moment :-)

Reply:
@jmertic - sorry for not doing this earlier in the week.  I've removed the sections that talks about conformance updates.

Reply:
Thanks @Joe-Winchester!

@swinslow See the updates below to base you change recommendations.